src/test/codegen/generateCode.test.ts

```diff
@@ -101,6 +103,116 @@ export const AuthorsPartition = LibraryTable.partition([AuthorModel, BookModel])
 `)
 })
 
+it("should generate multiple tables with simple models", () => {
+  const result = generateCode(`
+Tables:
+  Library:
+    Partitions:
+      Authors:
+        Author:
+          partitionKey: [Author, $id]
+          sortKey: [Author, $userId]
+          id: string
+          name: string
+          userId: string
+  AnotherLibrary:
```

Comment:
This reads a bit funky, how about we make `AnotherLibrary` => `Music` with a `Musician` model or something like that.